### PR TITLE
[CAPI/ACR] Reflect ACR reviews

### PR DESCRIPTION
**Pull request by zhoonit**
- [CAPI/ACR] Fix typo and doxygen tag

```
This patch reflects review from #1371, #1446

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

- [Trivial] Reflect acr review script

```
This patch reflect acr review scripts, which contains some trivial fixes
like adding a period at the end.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

- [CAPI/ACR] Add remark on tensor info getter

```
This patch adds remark on tensor info getter that info should not be
destroyed when fail to create

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Comment by zhoonit** [comment]
@p-czaja Updated as per comment, PTAL :)

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: api/capi/include/nntrainer.h [hunk 1]
```diff
@@ -436,12 +449,13 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
...
 /**
- * @brief Add data file to @a dataset
+ * @brief Adds data file to @a dataset.
  * @details Use this function to add a data file from where data is retrieved.
- * @remarks %http://tizen.org/privilege/mediastorage is needed if @a dataset is
- * saved to media storage.
- * @remarks %http://tizen.org/privilege/externalstorage is needed if @a model is
- * saved to external storage.
+ * @since_tizen 6.5
+ * @previliege %http://tizen.org/privilege/mediastorage is needed if @a dataset
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:455]: Simply put privilege name after '@privilege' tag: '@privilege %http://tizen.org/privilege/externalstorage' Note that currently tag name has typo ('@previliege' -> '@privilege')
The same applies to privileges in `ml_train_model_construct` function.

File: api/capi/include/nntrainer.h [hunk 2]
```diff
@@ -436,12 +449,13 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
...
+ * is saved to media storage.
+ * @previliege %http://tizen.org/privilege/externalstorage is needed if @a
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:457]: Simply put privilege name after '@privilege' tag: '@privilege %http://tizen.org/privilege/externalstorage' Note that currently tag name has typo ('@previliege' -> '@privilege')
The same applies to privileges in `ml_train_model_construct` function.

File: api/capi/include/nntrainer.h (hunk 1 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h:455]: the function does not always require privilege but for some cases it needs, do we put privilege tag in this case too?

Comment (p-czaja) [reply api/capi/include/nntrainer.h:455]: I think `privilege` tag should be added and cases when it is necessary should be described in `details`.

**Review by myungjoo — APPROVED** [review]

File: api/capi/include/nntrainer.h (hunk 1 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done :)

File: api/capi/include/nntrainer.h (hunk 2 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: done

File: api/capi/include/nntrainer.h [hunk 3]
```diff
@@ -79,13 +79,14 @@ typedef void *ml_train_dataset_h;
 /**
  * @brief Constructs the neural network model.
  * @details Use this function to create neural network model.
+ * Previlege is needed if @a model contains @c save_path pointing to either
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:82]: Typo: Previlege -> Privilege.

File: api/capi/include/nntrainer.h [hunk 4]
```diff
@@ -79,13 +79,14 @@ typedef void *ml_train_dataset_h;
...
+ * media storage or external storage.
  * @since_tizen 6.0
  * @remarks If the function succeeds, @a model must be released using
  * ml_train_model_destroy().
- * @remarks %http://tizen.org/privilege/mediastorage is needed if @a model is
- * saved to media storage.
- * @remarks %http://tizen.org/privilege/externalstorage is needed if @a model is
- * saved to external storage.
+ * @previlege %http://tizen.org/privilege/mediastorage
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:87]: Typo: `@previlege` -> `@privilege`.

File: api/capi/include/nntrainer.h [hunk 5]
```diff
@@ -79,13 +79,14 @@ typedef void *ml_train_dataset_h;
...
+ * @previlege %http://tizen.org/privilege/externalstorage
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:88]: Typo: `@previlege` -> `@privilege`.

File: api/capi/include/nntrainer.h [hunk 6]
```diff
@@ -451,11 +452,11 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
 /**
  * @brief Adds data file to @a dataset.
  * @details Use this function to add a data file from where data is retrieved.
+ * Previlege is needed if @a dataset is saved to either media storage or
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:455]: Typo: Previlege -> Privilege.

File: api/capi/include/nntrainer.h [hunk 7]
```diff
@@ -451,11 +452,11 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
...
+ * external storage.
  * @since_tizen 6.5
- * @previliege %http://tizen.org/privilege/mediastorage is needed if @a dataset
- * is saved to media storage.
- * @previliege %http://tizen.org/privilege/externalstorage is needed if @a
- * dataset is saved to external storage.
+ * @previlege %http://tizen.org/privilege/mediastorage
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:458]: Typo: `@previlege` -> `@privilege`.

File: api/capi/include/nntrainer.h [hunk 8]
```diff
@@ -451,11 +452,11 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
...
+ * @previlege %http://tizen.org/privilege/externalstorage
```
Comment (p-czaja) [inline api/capi/include/nntrainer.h:459]: Typo: `@previlege` -> `@privilege`.

File: api/capi/include/nntrainer.h (hunk 3 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done :)

File: api/capi/include/nntrainer.h (hunk 4 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 5 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 6 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 7 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 8 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

**Review by lhs8928 — APPROVED** [review]
LGTM

File: api/capi/include/nntrainer.h [hunk 9]
```diff
@@ -74,13 +79,14 @@ typedef void *ml_train_dataset_h;
 /**
  * @brief Constructs the neural network model.
  * @details Use this function to create neural network model.
+ * privilege is needed if @a model contains @c save_path pointing to either
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:82]: privilege -> Privilege?

File: api/capi/include/nntrainer.h [hunk 10]
```diff
@@ -223,12 +229,15 @@ int ml_train_model_set_dataset(ml_train_model_h model,
...
  * @details Use this function to get input tensors information of the model.
- * destroy @a info with @a ml_tensors_info_destroy() after use.
+ * destroy @a info with @c ml_tensors_info_destroy() after use.
+ * @since_tizen 6.5
  * @remarks @a model must be compiled before calling this function.
- * @remarks the returned @a info is newly created so it does not reflect future
- * changes in the model
+ * @remarks The returned @a info is newly created so it does not reflect future
+ * changes in the model.
+ * @remarks On returning error, info must shall not be destroyed with @c
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:239]: shall -> must.

File: api/capi/include/nntrainer.h [hunk 11]
```diff
@@ -242,12 +251,15 @@ int ml_train_model_get_input_tensors_info(ml_train_model_h model,
...
- * @brief Get output tensors information information of the model
+ * @brief Gets output tensors information of the model.
  * @details Use this function to get output tensors information of the model.
- * destroy @a info with @a ml_tensors_info_destroy() after use.
+ * destroy @a info with @c ml_tensors_info_destroy() after use.
+ * @since_tizen 6.5
  * @remarks @a model must be compiled before calling this function.
  * @remarks the returned @a info is newly created so it does not reflect future
  * changes in the model
+ * @remarks On returning error, info must shall not be destroyed with @c
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:261]: shall -> must

File: api/capi/include/nntrainer.h [hunk 12]
```diff
@@ -436,12 +450,13 @@ int ml_train_dataset_add_generator(ml_train_dataset_h dataset,
...
 
 /**
- * @brief Add data file to @a dataset
+ * @brief Adds data file to @a dataset.
  * @details Use this function to add a data file from where data is retrieved.
- * @remarks %http://tizen.org/privilege/mediastorage is needed if @a dataset is
- * saved to media storage.
- * @remarks %http://tizen.org/privilege/externalstorage is needed if @a model is
- * saved to external storage.
+ * privilege is needed if @a dataset is saved to either media storage or
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:455]: privilege -> Privilege

File: api/capi/include/nntrainer.h [hunk 13]
```diff
@@ -525,42 +540,40 @@ int ml_train_dataset_set_property_for_usage(ml_train_dataset_h dataset,
                                             ...);
 
 /**
- * @brief Save the model
- * @details Use this function to save the current model. @a format
+ * @brief Saves the model.
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:543]: This function saves the model?

File: api/capi/include/nntrainer.h [hunk 14]
```diff
@@ -525,42 +540,40 @@ int ml_train_dataset_set_property_for_usage(ml_train_dataset_h dataset,
...
+ * @details Use this function to save the current model. @a format.
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:544]: @a format~.~ ?

**Review by kparichay — APPROVED** [review]
LGTM other than some minor comments

File: api/capi/include/nntrainer.h (hunk 9 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Updated, thanks :)

File: api/capi/include/nntrainer.h (hunk 10 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 11 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 12 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

File: api/capi/include/nntrainer.h (hunk 14 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: Done

**Pull request merged**